mods/ts/rules/civilian-infantry.yaml

```diff
@@ -270,13 +272,16 @@ VISC_LRG:
 		HP: 500
 	Armament:
 		Weapon: SlimeAttack
+		FireDelay: 10
```

Comment:
doggie should really work, it does on TGR with Owner fixed.

visc_lrg should at least react to attacks, it might not attack on its own due to low scan radius.

I can't reproduce that issue with a new map either, sorry.